srcpkgs/melt/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'melt'
+pkgname=melt
+version=0.3.0
+revision=1
+create_wrksrc=yes
```

Comment:
the github-built release archive (`https://github.com/charmbracelet/melt/archive/refs/tags/v${version}.tar.gz`) has exactly the same contents, but doesn't need create_wrksrc

Reply:
indeed, but I prefer to use the one made and provided by the maintainers